src/aws-cpp-sdk-core/source/platform/linux-shared/OSVersionInfo.cpp

```diff
@@ -48,7 +48,7 @@ Aws::String ComputeOSVersionString()
     if(success >= 0)
     {
         Aws::StringStream ss;
-        ss << name.sysname << "/" << name.release;
```

Comment:
this was actually broken before too
`os/Darwin/23.6.0` -> `os/Darwin#23.6.0`
which is actually the correct format and actually how CLI has it in their user agent string
```
aws-cli/2.16.5 
  md/awscrt#0.20.11 
  ua/2.0 
  os/linux#5.10.230-202.885.amzn2int.x86_64 
  md/arch#x86_64 
  lang/python#3.11.8 
  md/pyimpl#CPython 
  cfg/retry-mode#standard 
  md/installer#exe 
  md/distrib#amzn.2 
  md/prompt#off 
  md/command#ec2.describe-instances
```